Djeddit/html/front_page.html

```diff
@@ -0,0 +1 @@
+<p>Front Page</p>
```

Comment:
Add a signin and signup link. This would also be a good time to implement Bootstrap into the project.